[WIP] Create Chapter on Research Ethics Committees Processes (#1843)
alan-turing-institute/the-turing-way: book/website/ethical-research/ethics-case-study.md

# Research Ethics Committees Processes

Research ethics committees (RECs), also known as institutional review boards (IRBs), exist within institutions to scrutinise the ethical implications of their research projects.

> RECs have historically focused on projects that involve human participants, meaning that they often prioritise the well-being of those involved in research projects.

Reviewer: ```suggestion
RECs have historically focused on projects that involve human participants, meaning that they often prioritise the well-being of those directly involved in research projects.
```

RECs are also a useful mechanism for the mitigation of reputational damage for their institutions, as they can ensure that research is conducted within the law.
Further, RECs are well-positioned to improve the state of scientific inquiry.

> This is because they can provide ethical oversight to expert researchers who do not have sufficient bandwidth to deeply reflect on the ethical questions their work raises.

Reviewer: ```suggestion
This is because they can provide ethical oversight to expert researchers who do not have sufficient experience or capacity to deeply reflect on the ethical questions their work raises.
```

>“any freely given, specific, informed and unambiguous indication of the data subject’s wishes by which he or she, by a statement or by a clear affirmative action, signifies agreement to the processing of personal data relating to him or her" ([ICO](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/consent/what-is-valid-consent/))

**Responding to this question** entails reflection on how data are sourced -- where they come from and who they pertain to.
Setting the legal discussions aside, this question requires following one of two approaches:
1. If employing data that is readily available (perhaps online or provided by a third party), it is important we understand the terms and conditions whereby those data are made available. The point here is to confirm that the data will be used following the purpose for which they were collected. In this respect, a useful subchapter in *The Turing Way* is that of [Data Licenses](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html?highlight=licen), which outlines different ways online data might be held.

> 2. If gathering new data through experiments with human research subjects, both personal information sheets and consent forms must be issued. On the one hand, personal information sheets provide research participants with all the information they need to know about taking part in the study. On the other hand, consent forms summarise the personal information sheet and provide a section for the potential participant to record their acceptance of the conditions. The consent form, in particular, should provide contact details for participants to reach out to the research team or institution (depending on internal processes) should they have any queries or concerns about the study in the future. The UK Data Service offer templates of both documents [online](https://www.ukdataservice.ac.uk/manage-data/legal-ethical/consent-data-sharing/consent-forms).

Reviewer: ```suggestion
2. If gathering new data through experiments with human research subjects, both participant information sheets (PISs) and consent forms must be issued. 
On the one hand, participant information sheets provide research participants with all the information they need to know about taking part in the study. 
On the other hand, consent forms summarise the participant information sheet and provide a section for the potential participant to record their acceptance of the conditions. 
The consent form, in particular, should provide contact details for participants to reach out to the research team or institution (depending on internal processes) should they have any queries or concerns about the study in the future. 
The UK Data Service offer templates of both documents [online](https://www.ukdataservice.ac.uk/manage-data/legal-ethical/consent-data-sharing/consent-forms): individual institutions often also make templates available for their researchers.
```

### Privacy and Security
>How will you ensure that any sensitive information your project uses is kept secure and made accessible only to pertinent members of the research team?

**Justification.** Privacy is a fundamental human right (see this [draft chapte](https://github.com/alan-turing-institute/the-turing-way/pull/1394/files?short_path=b95854e#diff-b95854e17574ab3b68f2f89991e4dddf199e01901dc8e235d87e2b0c8980a4d9) for *The Turing Way* for more on human rights).
Therefore, it is critical that projects be conducted in a way that respects privacy.

**Responding to this question** might require a review of your institution's internal policies.

> When handling personal data in particular, your institution might already have a data policy in place, as well as a process to conduct a data protection impact assessment (see [the ICO](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/accountability-and-governance/data-protection-impact-assessments/) for more details).

Reviewer: ```suggestion
When handling personal data in particular, your institution might already have a data policy in place, as well as a process to conduct a data protection impact assessment (see [the ICO](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/accountability-and-governance/data-protection-impact-assessments/) for more details).
Some funders also have policies for handling data collected or used in the course of research that they fund.
```

# Research Ethics Committees Processes

Research ethics committees (RECs), also known as institutional review boards (IRBs), exist within institutions to scrutinise the ethical implications of their research projects.

> RECs have historically focused on projects that involve human participants, meaning that they often prioritis the well-being of those involved in research projects.

Reviewer 2: ```suggestion
RECs have historically focused on projects that involve human participants, meaning that they often prioritise the well-being of those involved in research projects.
```

RECs are also a useful mechanism for the mitigation of reputational damage for their institutions, as they can ensure that research is conducted within the law.

> RECs are also well-positioned to improve the state of scientific inquiry.

Reviewer 2: These two sentences both begin "RECs are also", it's a bit clumsy to read.

Author: Sorted!

This is because they can provide ethical oversight to expert researchers who do not have sufficient bandwidth to deeply reflect on the ethical questions their work raises.

> This is not to liberate the researcher of their ethical responsibilities -- ultimately, they make the decisions as to how to conduct their work --, but to provide support in respect to these questions.

Reviewer 2: ```suggestion
This is not to liberate the researcher of their ethical responsibilities -- ultimately, they make the decisions as to how to conduct their work -- but to provide support in respect to these questions.
```

Not sure a comma is required here

Whilst there is a great deal of complexity involved in holding individuals accountable in scientific projects, this chapter assumes that RECs seek to improve how science is conducted at large.

There are many ways a research ethics committee might organise itself.
One common approach is to provide a form for all projects conducted at their institution to undergo ethical appraisal.
Below, we propose five headline questions that an ethics committee might want to know about. For each headline question, we provide:

> 1. Justification as to why it is important (why we suggect these headline questions to RECs), and

Reviewer 2: ```suggestion
1. Justification as to why it is important (why we suggest these headline questions to RECs), and
```

> 2. Certain considerations a researcher might make when preparing a response (what applicants shoul consider when reflecting on these points).

Reviewer 2: ```suggestion
2. Certain considerations a researcher might make when preparing a response (what applicants should consider when reflecting on these points).
```

### Consent
>How will you ensure any personal data collected for your research is done so with freely-given informed consent?

**Justification.** The purpose of ensuring consent for the collection, use or reuse of personal data is to respect the autonomy of individuals and operate in line with pertinent legislation.
It is also an important concept enshrined in law.
In the General Data Protection Regulation (GDPR), for example, *consent* is defined as:

>“any freely given, specific, informed and unambiguous indication of the data subject’s wishes by which he or she, by a statement or by a clear affirmative action, signifies agreement to the processing of personal data relating to him or her" ([ICO](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/consent/what-is-valid-consent/))

**Responding to this question** entails reflection on how data are sourced -- where they come from and who they pertain to.
Setting the legal discussions aside, this question requires following one of two approaches:

> 1. If employing data that is readily available (perhaps online or provided by a third party), it is important we understand the terms and conditions whereby those data are made available. The point here is to confirm that the data will be used following the purpose for which they were collected. In this respect, a useful subchapter in *The Turing Way* is that of *[Data Licenses](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html?highlight=licen)*, which outlines different ways online data might be held.

Reviewer 2: Previous internal (or external) links were not italicised, but "Data Licences" in this section was. I don't have a preference which way to go, just a request for consistency 😄

Author: Sorted! Thanks!

2. If gathering new data through experiments with human research subjects, both personal information sheets and consent forms must be issued. On the one hand, personal information sheets provide research participants with all the information they need to know about taking part in the study. On the other hand, consent forms summarise the personal information sheet and provide a section for the potential participant to record their acceptance of the conditions. The consent form, in particular, should provide contact details for participants to reach out to the research team or institution (depending on internal processes) should they have any queries or concerns about the study in the future. The UK Data Service offer templates of both documents [online](https://www.ukdataservice.ac.uk/manage-data/legal-ethical/consent-data-sharing/consent-forms).

### Privacy and Security
>How will you ensure that any sensitive information your project uses is kept secure and made accessible only to pertinent members of the research team?

> **Justification.** Privacy is a fundamental human right ([TTW](https://github.com/alan-turing-institute/the-turing-way/pull/1394/files?short_path=b95854e#diff-b95854e17574ab3b68f2f89991e4dddf199e01901dc8e235d87e2b0c8980a4d9)).

Reviewer 2: This "TTW" link is a bit obscure, it wasn't clear what I was clicking through to.

Author: Yes! Okay, I have a question: is it okay for me to link to a raft Turing Way chapter?

Reviewer 2: I _think_ so? @malvikasharan might have a better answer. I think it's nice though because maybe you could annotate the link saying "This is a draft chapter that needs review. If this is something you're interested in doing, please get in touch!"
